app/models/bookings/school.rb

```diff
@@ -177,6 +177,8 @@ class Bookings::School < ApplicationRecord
     end
   }
 
+  scope :onboarded, -> { joins(:profile) }
```

Comment:
I'm not sure if its redundant now, but the profile has a `completed?` method we may want to check as well?

https://github.com/DFE-Digital/schools-experience/blob/master/app/models/schools/school_profile.rb#L271

Reply:
True, it would be redundant to check for the `completed?` because the `schools/school_profile` model is only used in the onboarding process. The `bookings/profile` is the actual school profile which is only created when the onboarding is completed.